src/frontend/src/pages/TeamsPage/TeamSummary.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Card, Collapse, Col, Container, Row } from 'react-bootstrap';
+import { Team } from 'shared';
+import { useTheme } from '../../hooks/Theme.hooks';
+import { routes } from '../../utils/Routes';
+import { fullNamePipe, listPipe, wbsPipe } from '../../utils/Pipes';
+import styles from '../../stylesheets/pages/ProjectDetailPage/WorkPackageSummary.module.scss';
+import ReactMarkdown from 'react-markdown';
+import PageBlock from '../../layouts/PageBlock';
+
+const DESCRIPTION_WORD_LIMIT = 100;
+
+interface TeamSummaryProps {
+  team: Team;
+}
+
+const TeamSummary: React.FC<TeamSummaryProps> = ({ team }) => {
+  const [open, setOpen] = useState(false);
+  const theme = useTheme();
+
+  const membersList = listPipe(team.members, (t) => fullNamePipe(t));
+
+  const projectsList = team.projects.map((project, idx) => (
+    <>
+      <Link to={`${routes.PROJECTS}/${wbsPipe(project.wbsNum)}`}>{project.name}</Link>
+      {idx + 1 !== team.projects.length ? ', ' : ''}
+    </>
+  ));
+
+  let description = team.description;
+  let descriptionTooLong = false;
+  if (description.split(' ').length > DESCRIPTION_WORD_LIMIT) {
+    descriptionTooLong = true;
+    description = description.split(' ').slice(0, DESCRIPTION_WORD_LIMIT).join(' ') + '\u2026';
+  }
```

Comment:
This feels real janky to me. Personally, I'd suggest a character limit here if we're concerned about the length of the descriptions because that'd be pretty simple to implement, but if we wanna go with the word limit I think this bit of code needs to be a little cleaner/ we need to look at how descriptions are stored (also feels a bit weird to me, see comment above)

Reply:
so yes it is definitely incredibly janky. however, we can't do character limits because the descriptions are written in markdown which could cut off certain things like a link that is `[lasdf](sdfds)` halfway thorugh. This way still makes it cut off things like bold halfway through and turn it into ** but it's kinda better. Also I think cutting off half of words looks weird